projects/defi-basket/index.js

```diff
@@ -59,6 +59,12 @@ function polygonTVL() {
               token.asset.type === "IAaveV2Deposit"
             ) {
               tokens.push(token.asset.address);
+            } else if (token.asset.type === "IAaveRewards") {
+              let aTokenToAdd = token.asset.callParams.aToken;
+              tokens = [
```

Comment:
another way is

    if (!tokens.includes(aTokenToAdd))
      tokens.push(aTokenToAdd)

Reply:
(this is just a recommendation, you can ignore it :) , line 61 is the issue)

Reply:
That helps, thanks.